apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -179,9 +179,9 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
                             <p className="mb-1 -ml-2 inline px-2 py-1">
                               {getRecurringFreq({ t, recurringEvent: eventType.recurringEvent })}
                             </p>
-                            <input
+                            <TextField
                               type="number"
-                              min="1"
```

Comment:
This breaks the UI.
After:
![image](https://user-images.githubusercontent.com/1780212/220061503-8a0d85d8-e69a-44f8-aca2-1c216ec6a35e.png)
Before:
![image](https://user-images.githubusercontent.com/1780212/220061577-56ffc446-b150-4f86-b3ae-ba9dec6572f1.png)

Reply:
sure